package.json

```diff
@@ -29,12 +29,12 @@
     "fs-extra": "^1.0.0",
     "glob": "^7.1.2",
     "mkdirp": "^0.5.1",
-    "node-yaml": "^3.1.1",
+    "node-yaml": "^4.0.1",
     "rw": "^1.3.3",
     "tmp": "0.0.33",
     "webfonts-generator": "^0.4.0",
     "xml": "^1.0.1",
     "xml2js": "^0.4.19",
-    "yargs": "^5.0.0"
+    "yargs": "^16.2.0"
```

Comment:
Upgrading yargs seems to be a breaking change for "qwant-basic-gl-style" CI script that depends on this builder CLI:
https://github.com/Qwant/qwant-basic-gl-style/runs/2055635341

Reply:
Should the yargs config be updated to remain compatible, or should we update the script defined in qwant-basic-gl-style ?

Reply:
Thanks for checking that!

Like always when working on these two libs, I'm discovering a new level of messy things :smile: 

The CI is on `qwant-basic-gl-style`, but it runs a script defined half on `map-style-builder`, half in the CI params of `qwant-basic-gl-style`, to build this command `node test/index.js --style_dir  "../" "--icons=true"`.
This seems quite fragile already, but worked until now…

I think the problem is that this test CLI uses the same `yargs` parsing as the real command, just to get the args value, but doesn't pass required params `conf` and `style-dir` (it's `style_dir` here, for an obscure code reason, yet another mess…).
Yargs must have changed in the meantime by exiting when required params are not passed, which is a good thing in itself, but breaks our tests…
I'll look for a yargs option to restore the previous behavior temporarily for the tests.

But I'd really like to merge these two repos in the future, I see no point in maintaingin them separately whereas they have such strong dependencies.

Reply:
A first step could be to pin the version used on qwant-basic-gl-style CI.  
So that we can move forward on this repo, and clean up the whole process later ?

Reply:
I pushed a simple fix with a comment, it solves the CI issue https://github.com/Qwant/qwant-basic-gl-style/runs/2056515242
Better if we can avoid pinning the version, it's the kind of stuff that's easy to forget :)